packages/common/src/ui/components/loading/Biker/Biker.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { FC } from 'react';
+import { GlobalStyles, Typography, Box } from '@mui/material';
+
+const styles = {
+  '.biker': {
+    width: '106px',
+    height: '56px',
+    display: 'block',
+    margin: '30px auto',
+    backgroundImage:
+      'linear-gradient(#000 50px, transparent 0), linear-gradient(#000 50px, transparent 0), linear-gradient(#000 50px, transparent 0), linear-gradient(#000 50px, transparent 0), radial-gradient(circle 14px, #000 100%, transparent 0)',
+    backgroundSize: '48px 15px , 15px 35px, 15px 35px, 25px 15px, 28px 28px',
+    backgroundPosition: '25px 5px, 58px 20px, 25px 17px, 2px 37px, 76px 0px',
+    backgroundRepeat: 'no-repeat',
+    position: 'relative',
+    transform: 'rotate(-45deg)',
+    boxSizing: 'border-box',
+  },
+  '.biker::after, .biker::before': {
+    content: "''",
+    position: 'absolute',
+    width: '56px',
+    height: '56px',
+    border: '6px solid #000',
+    borderRadius: '50%',
+    left: '-45px',
+    top: '-10px',
+    backgroundRepeat: 'no-repeat',
+    backgroundImage:
+      'linear-gradient(#000 64px, transparent 0), linear-gradient(#000 66px, transparent 0), radial-gradient(circle 4px, #000 100%, transparent 0)',
+    backgroundSize: '40px 1px , 1px 40px, 8px 8px',
+    backgroundPosition: 'center center',
+    boxSizing: 'border-box',
+    animation: 'rotation 0.3s linear infinite',
+  },
+  '.biker::before': {
+    left: '25px',
+    top: '60px',
+  },
+
+  '@keyframes rotation': {
+    '0%': {
+      transform: 'rotate(0deg)',
+    },
+    '100%': {
+      transform: 'rotate(360deg)',
+    },
+  },
+} as const;
+
+const text = [
+  'One sec, I got hit by a bus...',
```

Comment:
I think if you put this out there, and ask for suggestions on telegram, we'd get 1,000 odd posts

Reply:
As it is, I was hitting refresh a dozen times trying to read them all 🤣

Reply:
That message is paying homage to our falling comrade William!

Look, the implementation isn't ideal but .. whatever!

Reply:
Oh. I did forget to mention that I tried using `NODE_ENV` and it doesn't work (Was going to put a boring spinner when not in dev!) so we'll have to setup some build thing for some dev variable, I think. But, I thought not to use translations as we'd use a 'real' (🙄 ) spinner in prod (Or, maybe just no text 🤣 )

Reply:
it's not like the translations work in prod environment anyway! 🤣 
I was going to look at `NODE_ENV` - had a feeling that wasn't working. I'll make an issue!